NewForce-Cohort-4/nutshell-the-canine-pirates
# Task Function MVP

# Description
Tasks MVP ticket. 
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
-

# Testing Instructions
1. Register an email
2. Log in with email
3. Click "New Task" button to render form
4. Fill out form and push "Save Task" 
5. Task should print to DOM
6. Click checkbox to mark a task as complete
7. The task should disappear from print when box is checked
8. Upon logout, the task cards should clear
# Checklist:
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
